docs/GETTING_STARTED.md

```diff
@@ -1,288 +1,225 @@
-`ctr version` and `ctr i ls`
+## Install soci-snapshotter
 
-**soci-snapshotter** - this you will need to make sure you can build from
-source. You should be able to run `make` in the soci-snapshotter dir.
-Some common issues:
-- getting the github repository `git clone
-https://github.com/awslabs/soci-snapshotter.git`
-- having the zlib developer tools (zlib is used for decompression and ztoc creation).
-Often fixed with `sudo yum install zlib-devel`
-- having the proper go version installed, as of this document please use 1.18+
+The soci-snapshotter project consists of 2 main components:
 
-**fuse** - confirm FUSE is installed for mounting userspace filesystems with `fusermount -V`
-(optional, enables mounting without root access)
+- `soci`: the CLI tool used to build/manage SOCI indices.
+- `soci-snapshotter-grpc`: the daemon (a containerd snapshotter plugin) used for lazy loading.
 
-## Start a local ORAS registry
-Most OCI registries do not currently support the "referrers" feature.
-SOCI relies on this feature to link its indices and manifests.  The [ORAS
-project](https://oras.land/) tries to fill this gap; in their words:
+Many `soci` CLI commands need to be run as `sudo`, because the metadata is saved
+in directories that a non-root user often does not have access to.
 
-> Registries are evolving as generic artifact stores. To enable this goal, the ORAS project provides a way to push and pull OCI Artifacts to and from OCI Registries.
+Currently to get the binaries, we need to build the project from source after cloing the repo:
 
-[The longer term solution for this issue is using the OCI Reference
-Types](https://github.com/awslabs/soci-snapshotter/issues/42).  For now we are
-relying on the ORAS project for the referrers feature.
+```shell
+$ git clone https://github.com/awslabs/soci-snapshotter.git
+$ cd soci-snapshotter
+$ make
+```
 
-Given this we want to start our registry using an OCI image from the ORAS 
-project.  We will run it on port 5000 as an HTTP (**NOT** HTTPS) server without
-certificates.  To do this we will use Docker:
+This builds the project binaries into the `./out` directory. You can install them
+to a `PATH` directory (`/usr/local/bin`) with:
 
-```
-docker run -d -p 5000:5000 --restart=always --name registry ghcr.io/oci-playground/registry:v3.0.0-alpha.1
+```shell
+$ sudo make install
 ```
 
-Since this is just a locally run HTTP server we can interact with
-it using standard `curl` commands. To confirm this is running lets
-use the following commands:
+## Push an image to your registry
 
-```
-docker ps
-```
+In this document we will use `rabbitmq` from DockerHub `docker.io/library/rabbitmq:latest`.
 
-which should yield a running container at 0.0.0.0:5000 for the ORAS image. Now
-to check that we can interact with the server run the following:
-```
-curl http://localhost:5000/v2/_catalog
+> You need to change `<AWS_ACCOUNT_ID>.dkr.ecr.<REGION>.amazonaws.com/rabbitmq` to your own registry.
```

Comment:
This first mention of ECR comes without warning. Can we get a short mention above, possibly with a link to a guide or doc on how to set up a repository and registry?

Reply:
fixed